js/main.js

```diff
@@ -1,2 +1,52 @@
 /* global data */
 /* exported data */
+var entries = [];
+var nextEntryId = 0;
+var $image = document.querySelector('.form-image');
+var $photoUrl = document.querySelector('#entry-url');
+var $form = document.querySelector('form');
+
+if(localStorage.getItem('entries') !== null) {
+ entries = JSON.parse(localStorage.getItem('entries'));
+ var biggest = entries[0].entryId;
+ for(var i = 0; i < entries.length; i++) {
+  if(entries[i].entryId > biggest) {
+    biggest = entries[i].entryId;
+  }
+ }
+ nextEntryId = biggest;
+}
```

Comment:
All localStorage code should be in `data.js` with your data model. Also, you should save the entire data model in localStorage, not just the entires. That will save you searching for the last entry, since you will have an up-to-date property named `nextEntryId` to tell you what id to use next.